src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -59,12 +60,23 @@ public DocumentExecuter(IDocumentBuilder documentBuilder, IDocumentValidator doc
         /// <see cref="IDocumentCache"/> and a set of <see cref="IConfigureExecutionOptions"/> instances.
         /// </summary>
         public DocumentExecuter(IDocumentBuilder documentBuilder, IDocumentValidator documentValidator, IComplexityAnalyzer complexityAnalyzer, IDocumentCache documentCache, IEnumerable<IConfigureExecutionOptions> configurations)
+            : this(documentBuilder, documentValidator, complexityAnalyzer, documentCache, configurations, new DefaultExecutionStrategySelector())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance with specified <see cref="IDocumentBuilder"/>,
```

Comment:
```suggestion
        /// Initializes a new instance with the specified <see cref="IDocumentBuilder"/>,
```

? I'm not a native speaker though.